app/javascript/components/lease_ups/LeaseUpApplicationsTable.js

```diff
@@ -87,52 +87,79 @@ const LeaseUpApplicationsTable = ({
       )
     },
     {
-      Header: 'Application Number',
+      Header: 'Application',
       accessor: 'application_number',
-      className: 'text-left',
+      headerClassName: 'non-resizable',
+      width: getCellWidth(96),
       Cell: (cell) => (
-        <Link
-          to={appPaths.toApplicationSupplementals(cell.original.application_id)}
-          className='has-border'
-        >
+        <Link to={appPaths.toApplicationSupplementals(cell.original.application_id)}>
           {cell.value}
         </Link>
       )
     },
-    { Header: 'First Name', accessor: 'first_name', Cell: resizableCell, className: 'text-left' },
-    { Header: 'Last Name', accessor: 'last_name', Cell: resizableCell, className: 'text-left' },
     {
-      Header: 'Accessibility Requests',
-      accessor: 'accessibility',
-      Cell: resizableCell,
-      className: 'text-left'
+      Header: 'First Name',
+      accessor: 'first_name',
+      headerClassName: 'non-resizable',
+      minWidth: getCellWidth(98),
+      Cell: textCell
+    },
+    {
+      Header: 'Last Name',
+      accessor: 'last_name',
+      headerClassName: 'non-resizable',
+      minWidth: getCellWidth(98),
+      Cell: textCell
     },
     {
       Header: 'HH',
       accessor: 'total_household_size',
-      Cell: resizableCell,
-      headerClassName: 'text-right td-min-narrow',
-      className: 'text-right td-min-narrow'
+      headerClassName: 'non-resizable',
+      width: getCellWidth(24)
+    },
+    {
+      Header: 'Requests',
+      accessor: 'accessibility',
+      headerClassName: 'non-resizable',
+      minWidth: getCellWidth(84),
+      Cell: textCell
     },
     {
       Header: 'Updated',
       accessor: 'status_last_updated',
-      headerClassName: 'text-right td-min-narrow',
-      className: 'text-right td-min-narrow',
+      headerClassName: 'non-resizable',
+      minWidth: getCellWidth(75),
       Cell: cellFormat.date
     },
     {
-      Header: 'Substatus',
+      Header: 'Latest Substatus',
       accessor: 'sub_status',
-      headerClassName: 'td-offset-right',
-      className: 'td-offset-right text-right td-min-wide'
+      className: 'td-offset-right',
+      headerClassName: 'td-offset-right non-resizable',
+      minWidth: getCellWidth(345),
+      Cell: textCell
     },
     {
-      Header: 'Lease Up Status',
+      Header: 'Status',
       accessor: 'lease_up_status',
-      headerClassName: 'td-min-wide tr-fixed-right',
-      className: 'td-min-wide td-status td-fixed-right',
-      Cell: (cell) => <LeaseUpStatusCell cell={cell} onChange={onLeaseUpStatusChange} />
+      paddingLeft: '10rem',
+      className: 'border-left padding-left td-status td-fixed-right',
+      headerClassName: 'border-left padding-left non-resizable tr-fixed-right',
+      minWidth: getCellWidth(192, true),
+      Cell: (cell) => {
+        const {
+          application_id: applicationId,
+          application_preference_id: preferenceId
```

Comment:
minor thing is that in my PR I remove the need for the application preference id, but I will deal with it after this is merged